src/types/input.types.ts

```diff
@@ -1,26 +1,41 @@
 import React from 'react';
-import { NativeSyntheticEvent, TextInputFocusEventData } from 'react-native';
+import { NativeSyntheticEvent, TextInputFocusEventData, ViewStyle } from 'react-native';
 
 export type InputProps = {
   type: 'default' | 'large';
   value: string;
   placeholder: string;
-  onChangeText: any;
+  onChangeText: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Change just to Function, cause the function that will get there is not always the setState function